api/src/Waitlist.ts

```diff
@@ -22,10 +27,10 @@ export class GearWaitlist {
    * console.log(waitlist);
    * ```
    */
-  async read(programId: Hex, messageId?: Hex): Promise<WaitlistType> {
+  async read(programId: Hex, messageId?: Hex): Promise<WaitlistItem[] | WaitlistItem> {
```

Comment:
Is there the ability to provide consistent return? I mean, `WaitlistItem[]` even for a single item.
It would be more convenient to deal with it on the frontend, if we will know that returned value is always an array.

Reply:
Override doesn't solve this problem?

Reply:
15-16 rows

Reply:
No, I meant like a simple check if there's a single `WaitlistItem` to still return it in an array with length 1